bedrock/base/templates/base-protocol.html

```diff
@@ -141,6 +141,11 @@
         {{ js_bundle('ui') }}
         {{ js_bundle('fxa') }}
         {{ js_bundle('data') }}
+        {% if switch('m24-navigation-and-footer') %}
```

Comment:
**Issue:** for the CSS we are using `{% if switch('m24-navigation-and-footer') and LANG.startswith('en-') %}`. Should the JS be limited to `en` also?